apps/linkedin-to-notion/src/contents/notion-oauth-nudge-wrapper-styling.ts

```diff
@@ -0,0 +1,19 @@
+import type { PlasmoCSConfig } from 'plasmo';
+
+import { waitForElementToExist } from './utils/utils';
+
+export const config: PlasmoCSConfig = {
+  matches: [
+    'https://www.notion.so/install-integration?response_type=code&client_id=06ba891a-ecd0-4622-8489-6dae87349cde*',
+  ],
+};
+
+(async () => {
+  await waitForElementToExist('div', 'Use a template provided by the developer');
+  // Wait 0.1s with a promise
```

Comment:
Why?

Reply:
Good question aha! If I remember well, it's because that's about the time I needed for the page to load properly and I didn't want to code another `waitForElement` for the `<div style="... grid-template-rows: 1fr 1fr; ...">`.

Reply:
Don't know if I answer your question though

Reply:
cf. call: do develop a waitForElem specific to the grid.

Reply:
Ok, turns out that `waitForElementToExist` and all the subsequent shenanigans weren't warranted after all